hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOMRatisSnapshots.java

```diff
@@ -692,11 +692,23 @@ public void testInstallIncrementalSnapshotWithFailure() throws Exception {
     // TODO: Add wait check here, to avoid flakiness.
 
     // Verify the metrics
-    DBCheckpointMetrics dbMetrics = leaderOM.getMetrics().
-        getDBCheckpointMetrics();
-    assertEquals(0, dbMetrics.getLastCheckpointStreamingNumSSTExcluded());
-    assertTrue(dbMetrics.getNumIncrementalCheckpoints() >= 1);
-    assertTrue(dbMetrics.getNumCheckpoints() >= 3);
+    GenericTestUtils.waitFor(() -> {
```

Comment:
It seems like this change is meant to address the "todo" comment listed above:  https://github.com/apache/ozone/blob/2a14794d916e9e667a2ff073c8360db39c47f576/hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOMRatisSnapshots.java#L688-L693

If so, please remove that comment.

Reply:
done